packages/astro/src/default/styles/markdown.css

```diff
@@ -9,7 +9,8 @@
   --code-border-radius: 4px;
   --code-border-width: 1px;
   --code-box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.1), inset 0 1px 0 0 rgba(255, 255, 255, 0.5);
-  --link-color: var(--tk-text-accent);
+  --link-color: var(--tk-elements-link-primaryColor);
+  --link-color-hover: var(--tk-elements-link-primaryColorHover);
```

Comment:
Oh looks like in the code the right name is used?

Reply:
I'm not sure I'm following, but in the code below the `--link-color` is used. Previously this was using just generic `--tk-text-accent` color, now it's using link token. Related to https://github.com/stackblitz/tutorialkit/pull/202#discussion_r1703952464.

Reply:
Ah no worries, I didn't include enough context. 🙏 

I was referring to the fact that this is using `primaryColorHover` and not `primaryHoverColor`.